libs/ConfigKit/source/Config.cpp

```diff
@@ -0,0 +1,38 @@
+// Leka - LekaOS
+// Copyright 2021 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstddef>
+#include <cstdint>
+#include <cstdio>
+#include <span>
+
+#include "ConfigKit.h"
+#include "LogKit.h"
+
+using namespace leka;
+
+auto ConfigKit::read(Config const &config) -> uint8_t
+{
+	_file.open(config.path(), "r");
+	if (!_file.is_open()) {
+		return config.default_value();
+	}
+	auto input = std::array<uint8_t, 1> {};
+	_file.read(input);
```

Comment:
Question bête peut-être, pourquoi avoir utilisé un `std::array` plutôt que directement un `uint8_t`?

```suggestion
	auto input = uint8_t {0};
	_file.read(input, 1);
```

Reply:
> 

La fonction prend un `std::span `en paramètre. Dans les tests, je n'utilisais que des pointeurs ou des `std::array` pour les fonctions `read` et `write`.